src/modules/create-market/helpers/build-create-market.js

```diff
@@ -0,0 +1,51 @@
+import { augur } from 'services/augurjs'
+import speedomatic from 'speedomatic'
+import { BINARY, CATEGORICAL, SCALAR } from 'modules/markets/constants/market-types'
+import { DESIGNATED_REPORTER_SELF } from 'modules/create-market/constants/new-market-constraints'
+
+export const buildCreateMarket = (newMarket, isEstimate, universe, loginAccount, contractAddresses) => {
+  const tags = []
+  if (newMarket.tag1) tags.push(newMarket.tag1)
+  if (newMarket.tag2) tags.push(newMarket.tag2)
+  // General Properties
```

Comment:
Possibly a more informative label: `Properties common to all markets`

Reply:
was copied and pasted from existing code, will update to make more informative